src/factory/src/state.rs

```diff
@@ -50,10 +53,36 @@ impl Default for State {
 }
 
 pub fn get_token_bytecode() -> &'static [u8] {
-    include_bytes!("token.wasm")
+    &[]
```

Comment:
What do we need this function for? It shouldn't be used anywhere, right?

Reply:
it should change, we just need this first: https://github.com/infinity-swap/ic-helpers/pull/31